libs/ui/src/hooks/useRefreshToken/useRefreshToken.test.ts

```diff
@@ -52,12 +30,10 @@ describe('useRefreshToken', () => {
                 new Error(failedResponse.statusText, {cause: failedResponse})
             );
 
-            expect(localStorageMock.setItem).not.toHaveBeenCalled();
             expect(fetchMock).toHaveBeenCalledTimes(1);
             expect(fetchMock).toHaveBeenCalledWith('/auth/refresh', {
-                body: '{}',
                 headers: {'Content-Type': 'application/json'},
-                method: 'POST'
+                method: 'GET'
```

Comment:
**question:** Qu'est-ce qui a motivé la décision de modifier le verbe HTTP  de POST à GET ?

Reply:
On ne passe pas de body et techniquement il n'y a pas d'entrée en base de donnée

Reply:
[Préambule : je ne vais pas poser de véto pour ça]
Je pense bien que le requête devrait être un POST car on ne récupère pas une ressource mais on la crée bien.
Elle n'est pas stockée mais elle est bien générée par le backend.
Ç'aurait été un GET si le refresh token était stockée en base et qu'on le renvoyait dans la requête.

J'ai demandé à ChatGTP pour avoir son avis : 

<img width="762" alt="Capture d’écran 2024-08-02 à 16 05 34" src="https://github.com/user-attachments/assets/96fe6bb9-243b-4774-af1d-7c209ea3cf61">

Reply:
Dites moi ce que vous en pensez.

Reply:
Oui c'est vrais que cela fait sens. On est revenu en arrière ducoup 👍